packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -0,0 +1,671 @@
+        },
+        {
+            "fqdn": "login.sigmanetcorp.us",
+            "values": [
+                "${yourOktaDomain}.customdomains.okta1.com"
+            ],
+            "recordType": "CNAME"
+        }
+    ],
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "DELETE"
+                ]
+            }
+        },
+        "verify": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3/verify",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+### Verify Domain
+
+<ApiOperation method="post" url="/api/v1/domains/{id}/verify" />
+
+Verifies the domain.
+
+#### Request path parameters
+
+Verifies the domain with given `id`.
+
+ Parameter  | Description  | Param Type | DataType | Required |
+| --------- | ------------ | ---------- | -------- | -------- |
+| id        | `id`         | URL        | String   | TRUE     |
+
+
+#### Request query parameters
+N/A
+
+#### Request body
+N/A
+
+#### Response body
+
+The [DomainResponse](#domainresponse-object)
+
+
+#### Usage examples
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/domains/{id}/verify"
+```
+
+##### Response
+```json
+{
+    "id": "OcDz6iRyjkaCTXkdo0g3",
+    "domain": "login.example.com",
+    "validationStatus": "VERIFIED",
+    "dnsRecords": [
+        {
+            "fqdn": "_oktaverification.login.example.com",
+            "values": [
+                "79496f234c814638b1cc44f51a782781"
+            ],
+            "recordType": "TXT"
+        },
+        {
+            "fqdn": "login.example.com",
+            "values": [
+                "${yourOktaDomain}.customdomains.okta1.com"
+            ],
+            "recordType": "CNAME"
+        }
+    ],
+   "_links": {
+           "certificate": {
+               "href": "http://rain.okta1.com:1802/api/v1/domains/OcDz6iRyjkaCTXkdo0g3/certificate",
+               "hints": {
+                   "allow": [
+                       "PUT"
+                   ]
+               }
+           },
+           "self": {
+               "href": "http://rain.okta1.com:1802/api/v1/domains/OcDz6iRyjkaCTXkdo0g3",
+               "hints": {
+                   "allow": [
+                       "GET",
+                       "DELETE"
+                   ]
+               }
+           }
+   }
+}
+```
+### Create certificate
+
+<ApiOperation method="put" url="/api/v1/domains/{id}/certificate" />
+
+Create certificate for the domain.
+
+#### Request path parameters
+
+Creates certificate for the domain with given `id`.
+
+ Parameter  | Description  | Param Type | DataType | Required |
```

Comment:
add the DataType information and Required information in the Description, please!  :)

```suggestion
 Parameter  | Type | Description |
```